Baking Converter/ConverterViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  ConverterViewController.swift
+//  Baking Converter
+//
+//  Created by Michael Johnson on 6/10/17.
+//  Copyright © 2017 Michael Johnson. All rights reserved.
+//
+
+import UIKit
+import Foundation
+
+class ConverterViewController: UIViewController {
+
+    @IBOutlet weak var ingredientsPicker: UIPickerView!
+    @IBOutlet weak var inputUnitsPicker: UIPickerView!
+    @IBOutlet weak var outputUnitsPicker: UIPickerView!
+    @IBOutlet weak var inputField: UITextField!
+    @IBOutlet weak var resultsLabel: UILabel!
```

Comment:
I think they changed it so that they should no longer be `weak. [Source](https://stackoverflow.com/a/34212435/112862)

Reply:
Fixed.